include/core/lethegridtools.h

```diff
@@ -0,0 +1,150 @@
+//
+// Created by lucka on 2021-07-15.
+//
```

Comment:
Add license information.
Also remove author name. I am trying to move out of "file belonging to someone" mindset.